docs/pages/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.mdx

```diff
@@ -0,0 +1,306 @@
+---
+title: "AWS Route 53 GSLB Multi-Region Proxy Peering High Availability Deployment Guide"
```

Comment:
This title has a lot of jargon in it. Can we simplify it a bit?

Reply:
Would "AWS Multi-Region Single Cluster Deployment" work? I don't know if we need HA explicitly stated in the title.

Reply:
I think having HA in the title is important.

AWS Multi-Region High Availability Deployment Guide

Reply:
Updated title